setup.sh

```diff
@@ -64,3 +69,6 @@ case "${distribution}-${code_name}" in
     ;;
 esac
 
+if [ -f "apache-arrow-apt-source-latest-${code_name}.deb" ]; then
+  rm "apache-arrow-apt-source-latest-${code_name}.deb"
+fi
```

Comment:
Why do you remove this here instead of after L31?

Reply:
FIx: [e62ae2f](https://github.com/groonga/groonga/pull/1710/commits/e62ae2ff54c9267aeefec2906f6c7feaa707d25d)